vars/starter.yml

```diff
@@ -0,0 +1,27 @@
+---
+drupal_build_composer_project: true
+
+# None necessary, should all be described in the given project.
+#drupal_composer_dependencies: []
+
+# Presently pointing at the main branch.
+drupal_composer_project_package: 'islandora/islandora-starter-site:dev-main'
+
+# XXX: Strictly, irrelevant due to using the `--existing-config` flag.
+drupal_install_profile: minimal
+
+# None necessary, due to using an existing configuration.
+#drupal_enable_modules: []
+
+drupal_site_install_extra_args:
+  - '--existing-config'
+
+openseadragon_composer_require: false
```

Comment:
Some of these - setting up Matomo and FITS - seem like they're `false` only because theyre out of scope at the moment but we probably eventually want them. Can you denote/comment/group them in some way to distinguish from the others which we do/don't need due to the architecture of the Starter Site?

Reply:
Took a shot in https://github.com/Islandora-Devops/islandora-playbook/pull/226/commits/52f962aa6aad7901370e2d634c40991c51efe198 ... thoughts?

Reply:
I feel like I should mention: Anything that _does_ need to do any configuration probably maybe shouldn't do it by `config:set`'ing anything, but instead to use other mechanisms such as `config_override` to avoid potentially dirtying configs... to avoid specific environmental concerns creeping into things; for example: If the `matomo.settings:url_http` value was made to dynamically point at the hostname instead of just specifying `localhost:8000`, then yeah, it wouldn't really belong in the config that gets used between environments where that remote would vary.

... This... _does_ kind of get into a weird area with [how the JWT stuff is presently configured](https://github.com/Islandora-Devops/islandora-playbook/blob/311aa2126947869ce20d7765e9689de3d219d835/roles/internal/webserver-app/tasks/jwt.yml#L10-L39), in that because things were based on the "standard" playbook install, that it just so happens to match the paths; however, if other paths are ever specified for `webserver_app_jwt_key_path`, then it could end up diverging...